experimental/dds/tree2/src/feature-libraries/sequence-field/format.ts

```diff
@@ -404,6 +404,16 @@ export type Detach<TNodeChange = NodeChangeType> =
 export const Detach = <Schema extends TSchema>(tNodeChange: Schema) =>
 	Type.Union([DeleteSchema(tNodeChange), MoveOut(tNodeChange), ReturnFrom(tNodeChange)]);
 
+export interface MovePlaceholder<TNodeChange>
+	extends CellTargetingMark,
+		HasRevisionTag,
+		HasMoveId,
+		HasChanges<TNodeChange>,
+		CellTargetingMark {
```

Comment:
It extends `CellTargetingMark` twice. :)
```suggestion
	extends CellTargetingMark,
		HasRevisionTag,
		HasMoveId,
		HasChanges<TNodeChange> {
```